### PR TITLE
docs: Support for Kannada Translations (KAN_IN)

### PR DESCRIPTION
**Pull request by supminn**
I have added the Kannada translation for SECURITY.md and remote-schema.md files.
I have updated the README file with the links to their references.

P.S: Both the documents have been proof read and verified.

### CODE REVIEW

File: translations/SECURITY.kannada.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- ಸುರಕ್ಷತೆಗಾಗಿ ಹಸುರಾ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಂಜಿನ್‌ನ ಘಟಕಗಳನ್ನು ಟ್ಯೂನಿಂಗ್ ಮಾಡಲು ನಿಮಗೆ ಸಹಾಯ ಬೇಕಾಗುತ್ತದೆ.
+- ಭದ್ರತಾ ನವೀಕರಣಗಳನ್ನು ಅನ್ವಯಿಸಲು ನಿಮಗೆ ಸಹಾಯದ ಅಗತ್ಯವಿದೆ.
+- ನಿಮ್ಮ ಸಮಸ್ಯೆ ಭದ್ರತೆಗೆ ಸಂಬಂಧಿಸಿಲ್ಲ.
+
+## ಭದ್ರತಾ ದುರ್ಬಲತೆಯ ಪ್ರತಿಕ್ರಿಯೆ
+
+ಪ್ರತಿ ವರದಿಯನ್ನು 3 ಕೆಲಸದ ದಿನಗಳಲ್ಲಿ ಯೋಜನೆಯ ನಿರ್ವಾಹಕರು ಮತ್ತು ಭದ್ರತಾ ತಂಡವು ಸ್ವೀಕರಿಸುತ್ತದೆ ಮತ್ತು ವಿಶ್ಲೇಷಿಸುತ್ತದೆ.
+
+ಸಮಸ್ಯೆಯ ವಿಶ್ಲೇಷಣೆ ಮತ್ತು ಪರಿಹಾರದ ಪ್ರತಿ ಹಂತದಲ್ಲೂ ವರದಿಗಾರರನ್ನು ನವೀಕರಿಸಲಾಗುತ್ತದೆ. (ಟ್ರಯೇಜ್ -> ಸರಿ -> ಪ್ರಕಟಿಸಿ)
+
```
Comment (harishbala1o) [inline translations/SECURITY.kannada.md:25]: (ಟ್ರಯೇಜ್ -> ಸರಿ -> ಪ್ರಕಟಿಸಿ) can be changed to (ಟ್ರಯೇಜ್ -> ಫಿಕ್ಸ್ -> ರಿಲೀಸ್)

File: translations/remote-schemas.kannada.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# ರಿಮೋಟ್ ಸ್ಕೀಮಾಗಳು
+
+ನೀವು ಎಲ್ಲಾ ಪ್ರಕಾರದ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಅನ್ನು ಒಂದೇ ಸ್ಥಳದಿಂದ ಪ್ರಶ್ನಿಸಲು ಬಯಸಿದರೆ ನಂತರ ದೂರಸ್ಥ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸ್ಕೀಮಾವನ್ನು ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಂಜಿನ್‌ನ ಪೋಸ್ಟ್‌ಗ್ರೆಸ್ ಆಧಾರಿತ ಸ್ಕೀಮಾದೊಂದಿಗೆ ವಿಲೀನಗೊಳಿಸಿ. ರಿಮೋಟ್ ಸ್ಕೀಮಾಗಳು ಕೆಲವು ಉದಾಹರಣೆಗಳಿಗೆ ಸೂಕ್ತವಾಗಿವೆ:
```
Comment (harishbala1o) [inline translations/remote-schemas.kannada.md:3]: This sentence requires improved translation to provide the same understanding as the English version.
 
 This preceding sentence contains no information on (Hasura has the ability to merge remote GraphQL schemas and provide a unified GraphQL API.)

File: translations/remote-schemas.kannada.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+- ರೂಪಾಂತರಗಳನ್ನು ಉತ್ತಮಗೊಳಿಸುವುದು (ಉದಾಹರಣೆಗೆ ಒಳಪಡಿಸುವ ಮೊದಲು ಮೌಲ್ಯೀಕರಣ)
```
Comment (harishbala1o) [inline translations/remote-schemas.kannada.md:5]: ರೂಪಾಂತರಗಳನ್ನು provides different meaning.

File: translations/remote-schemas.kannada.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+- ಪಾವತಿಗಳು, ಇತ್ಯಾದಿ ವೈಶಿಷ್ಟ್ಯಗಳನ್ನು ಬೆಂಬಲಿಸುವುದು ಮತ್ತು ಅವುಗಳನ್ನು ಪ್ರವೇಶಿಸಲು ಸ್ಥಿರವಾದ ಇಂಟರ್ಫೇಸ್ ಅನ್ನು ಒದಗಿಸುವುದು ಅಂದರೆ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಂಜಿನ್‌ನ ಹಿಂದಿನ ಎಪಿಐ
```
Comment (harishbala1o) [inline translations/remote-schemas.kannada.md:6]: ಅಂದರೆ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಂಜಿನ್‌ನ ಹಿಂದಿನ ಎಪಿಐ provides different meaning.

File: translations/remote-schemas.kannada.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+- ಇತರ ಮೂಲಗಳಿಂದ ಡೇಟಾವನ್ನು ಬೇರ್ಪಡಿಸುವುದು (_ಉದಾಹರಣೆಗೆ ಹವಾಮಾನ ಎಪಿಐ ಅಥವಾ ಇನ್ನೊಂದು ಡೇಟಾಬೇಸ್‌ನಿಂದ_)
+
+ಕಸ್ಟಮ್ ವ್ಯವಹಾರ ತರ್ಕವನ್ನು ಬೆಂಬಲಿಸಲು, ನೀವು ಕಸ್ಟಮ್ ಸರ್ವರ್ ಅನ್ನು ರಚಿಸಬೇಕಾಗುತ್ತದೆ (ನೋಡಿ [ಬಾಯ್ಲರ್‌ಪ್ಲೇಟ್‌ಗಳು](../ಸಮುದಾಯ/ಬಾಯ್ಲರ್‌ಪ್ಲೇಟ್‌ಗಳು/ರಿಮೋಟ್-ಸ್ಕೀಮಾಸ್)) ಮತ್ತು ಅದರ ಸ್ಕೀಮಾವನ್ನು ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಂಜಿನ್‌ನೊಂದಿಗೆ ವಿಲೀನಗೊಳಿಸಿ.
```
Comment (harishbala1o) [inline translations/remote-schemas.kannada.md:9]: ನೀವು ಕಸ್ಟಮ್ ಸರ್ವರ್ ಅನ್ನು ರಚಿಸಬೇಕಾಗುತ್ತದೆ needs to be changed to ನೀವು ಕಸ್ಟಮ್ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್ ಅನ್ನು ರಚಿಸಬೇಕಾಗುತ್ತದೆ

File: translations/remote-schemas.kannada.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+[![video demo of merging remote schemas](https://img.youtube.com/vi/eY4n9aPsi0M/0.jpg)](https://www.youtube.com/watch?v=eY4n9aPsi0M)
+
+[ರಿಮೋಟ್ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸ್ಕೀಮಾವನ್ನು ವಿಲೀನಗೊಳಿಸಿ (YouTube link)](https://youtu.be/eY4n9aPsi0M)
+
+## ನಾವೀಗ ಆರಂಭಿಸೋಣ
+
+ರಿಮೋಟ್ ಸ್ಕೀಮಾವನ್ನು ಪ್ರಯತ್ನಿಸಲು ಹೀರೋಕು ವೇಗವಾದ ಮಾರ್ಗವಾಗಿದೆ.
+
+1. ಉಚಿತ ಪೋಸ್ಟ್‌ಗ್ರೆಸ್‌ನೊಂದಿಗೆ ಹೀರೋಕು ನಲ್ಲಿ ಎಂಜಿನ್ ಅನ್ನು ನಿಯೋಜಿಸಲು, ಈ ಕೆಳಗಿನ ಬಟನ್ ಅನ್ನು ಕ್ಲಿಕ್ ಮಾಡಿ:
```
Comment (harishbala1o) [inline translations/remote-schemas.kannada.md:23]: ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಂಜಿನ್

File: translations/remote-schemas.kannada.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+   [![Deploy to Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/hasura/graphql-engine-heroku)
+
+2. ಹಸುರಾ ಕನ್ಸೋಲ್ ತೆರೆಯಿರಿ
+
+   ನಿರ್ವಾಹಕ ಕನ್ಸೋಲ್ ತೆರೆಯಲು `https: // <app-name> .herokuapp.com` (_ ನಿಮ್ಮ ಅಪ್ಲಿಕೇಶನ್‌ನ ಹೆಸರಿನೊಂದಿಗೆ _ <app-name \> ಬದಲಾವಣೆ) ಗೆ ಹೋಗಿ.
```
Comment (harishbala1o) [inline translations/remote-schemas.kannada.md:29]: ಬದಲಾವಣೆ needs to be ಬದಲಾಯಿಸಿ

File: translations/remote-schemas.kannada.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+3. ನಿಮ್ಮ ಮೊದಲ ರಿಮೋಟ್ ಸ್ಕೀಮಾವನ್ನು ವಿಲೀನಗೊಳಿಸಿ ಮತ್ತು ಅದನ್ನು ಪ್ರಶ್ನಿಸಿ
+
+   ನಿರ್ವಾಹಕ ಕನ್ಸೋಲ್‌ನಲ್ಲಿ `Remote Schemas` ಟ್ಯಾಬ್ ತೆರೆಯಿರಿ ಮತ್ತು ` Add` ಬಟನ್ ಮೇಲೆ ಕ್ಲಿಕ್ ಮಾಡಿ. ಕೆಳಗಿನ ವಿವರಗಳನ್ನು ಭರ್ತಿ ಮಾಡಿ:
+
+   - Remote Schema ಹೆಸರು: `countries` (_ಈ ರಿಮೋಟ್ ಸ್ಕೀಮಾಗೆ ಅಲಿಯಾಸ್ _).
+   - ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್ URL: `https://countries.trevorblades.com/` (_एಈ ವೈಶಿಷ್ಟ್ಯವನ್ನು ತ್ವರಿತವಾಗಿ ಪರೀಕ್ಷಿಸಲು ನಾವು ಬಳಸುವ ಸಾರ್ವಜನಿಕ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಪಿಐ;[@trevorblades](https://github.com/trevorblades) ನಿರ್ವಹಿಸಿದವರು._
+   - ಉಳಿದ ಕಾನ್ಫಿಗರೇಶನ್ ಸೆಟ್ಟಿಂಗ್‌ಗಳನ್ನು ನಿರ್ಲಕ್ಷಿಸಿ ಮತ್ತು `Add Remote Schema` ಬಟನ್ ಮೇಲೆ ಕ್ಲಿಕ್ ಮಾಡಿ.
+
+   ``GraphiQL` ಟ್ಯಾಬ್ ಮಾಡಿ ಮತ್ತು ಈ ಕೆಳಗಿನ ಪ್ರಶ್ನೆಯನ್ನು ಚಲಾಯಿಸಿ (ಎಡಭಾಗದಲ್ಲಿರುವ ಪ್ರಶ್ನೆ ವಿಂಡೋದಲ್ಲಿ _ ಅಂಟಿಸಿ ಮತ್ತು _ ▶️ _(play) ಬಟನ್ ಮೇಲೆ ಕ್ಲಿಕ್ ಮಾಡಿ _):
```
Comment (harishbala1o) [inline translations/remote-schemas.kannada.md:39]: ``GraphiQL` ಟ್ಯಾಬ್ ಮಾಡಿ ಮತ್ತು ಈ ಕೆಳಗಿನ ಪ್ರಶ್ನೆಯನ್ನು ಚಲಾಯಿಸಿ needs to changed

File: translations/remote-schemas.kannada.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+ಹೆಚ್ಚಿನ ಭಾಷೆಗಳು, ಚೌಕಟ್ಟುಗಳು, ಸರ್ವರ್‌ಲೆಸ್ ಪ್ಲಾಟ್‌ಫಾರ್ಮ್‌ಗಳು ಇತ್ಯಾದಿಗಳಿಗಾಗಿ ಬಾಯ್ಲರ್‌ಪ್ಲೇಟ್‌ಗಳನ್ನು ಪುನರಾವರ್ತಿಸಲಾಗುತ್ತಿದೆ ಮತ್ತು ಸಮುದಾಯದ ಕೊಡುಗೆಗಳು ತುಂಬಾ ಸ್ವಾಗತಾರ್ಹ ಎಂಬುದನ್ನು ದಯವಿಟ್ಟು ಗಮನಿಸಿ.
+
+## ಎಚ್ಚರಿಕೆಗಳು
+
+**ಪ್ರಸ್ತುತ ಮಿತಿಗಳು**:
+
+- ಪ್ರಕಾರದ ಹೆಸರುಗಳು ಮತ್ತು ನೋಡ್ ಹೆಸರುಗಳು ಎಲ್ಲಾ ವಿಲೀನಗೊಂಡ ಸ್ಕೀಮಾಗಳಾದ್ಯಂತ ಅನನ್ಯವಾಗಿರಬೇಕು (ಕೇಸ್-ಸೆನ್ಸಿಟಿವ್ ಮ್ಯಾಚಿಂಗ್). ಮುಂದಿನ ಕೆಲವು ಪುನರಾವರ್ತನೆಗಳಲ್ಲಿ, ನಿಖರವಾದ ಹೆಸರುಗಳು ಮತ್ತು ರಚನೆಗಳೊಂದಿಗೆ ವಿಲೀನ ವಿಧಾನಗಳಿಗೆ ಬೆಂಬಲ ಲಭ್ಯವಿರುತ್ತದೆ.
+- ವಿಭಿನ್ನ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್‌ಗಳಿಂದ ನೋಡ್‌ಗಳನ್ನು ಒಂದೇ ಪ್ರಶ್ನೆ/ಮ್ಯುಟೇಶನ್‌ನಲ್ಲಿ ಬಳಸಲಾಗುವುದಿಲ್ಲ. ಎಲ್ಲಾ ಉನ್ನತ ಮಟ್ಟದ ನೋಡ್‌ಗಳು ಒಂದೇ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್‌ನಿಂದ ಇರಬೇಕು.
+- ರಿಮೋಟ್ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್‌ನಲ್ಲಿನ ಚಂದಾದಾರಿಕೆಗಳು ಬೆಂಬಲಿತವಾಗಿಲ್ಲ. ರಿಮೋಟ್ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್‌ಗಳಲ್ಲಿ ಚಂದಾದಾರಿಕೆಗಳನ್ನು ಬೆಂಬಲಿಸುವುದಿಲ್ಲ.
```
Comment (harishbala1o) [inline translations/remote-schemas.kannada.md:71]: Sentence is repeated twice

File: translations/remote-schemas.kannada.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+## ದಸ್ತಾವೇಜನ್ನು
+
+ಪೂರ್ಣ ಓದಿ [ದಸ್ತಾವೇಜನ್ನು](https://hasura.io/docs/latest/graphql/core/remote-schemas/index.html).
+
+## ಅನುವಾದಿಸು
+
+ಈ ಡಾಕ್ಯುಮೆಂಟ್ ಈ ಕೆಳಗಿನ ಅನುವಾದಗಳಲ್ಲಿ ಲಭ್ಯವಿದೆ:
+
+- [Hindi :in:](translations/remote-schemas.hindi.md)
```
Comment (harishbala1o) [inline translations/remote-schemas.kannada.md:83]: French version can be linked here 
https://github.com/hasura/graphql-engine/blob/master/translations/remote-schemas.french.md

File: translations/remote-schemas.kannada.md (hunk 3 above)
Comment (supminn) [reply translations/remote-schemas.kannada.md:5]: ರೂಪಾಂತರಗಳನ್ನು would provide the meaning for `transformation` (ರೂಪಾಂತರ -- to transform)
We could replace it by ಬದಲಾವಣೆ, ಪರಿವರ್ತನೆ, ಮಾರ್ಪಾಟು.
Let me know which is suitable.

File: translations/remote-schemas.kannada.md (hunk 2 above)
Comment (supminn) [reply translations/remote-schemas.kannada.md:3]: Thank you for your review comments @harishbala1o 
I have incorporated the necessary changes. Do let me know if there is anything else that requires rephrasing.

File: translations/remote-schemas.kannada.md (hunk 3 above)
Comment (harishbala1o) [reply translations/remote-schemas.kannada.md:5]: Mutations is the name of a GrapghQL operation, so it makes no sense to mention the translated word.

ಮ್ಯುಟೇಷನ್ಸ್ಗಳನ್ನು  ಕಸ್ಟಮೈಸ್ ಮಾಡುವುದು would provide exact meaning

**Comment by supminn** [comment]
Hi @harishbala1o 
Thanks for suggesting the corrections. I have incorporated the changes.

**Comment by supminn** [comment]
Hi @harishbala1o 
Are there any more suggestions? If not, could you please approve the PR?

**Review by harishbala1o — APPROVED** [review]

**Comment by Stefmore02** [comment]
Hey! @supminn 

We truly appreciate your contribution to Hasura OSS during Hacktoberfest 2022! We'd love to send you some Hasura swag as a token of appreciation. Could you please send me an email to [estefani@hasura.io](mailto:estefani@hasura.io) with a reference of this PR so I can send you the swag form? ✨

**Pull request closed without merging**